templates/components/scenarios.tmpl

```diff
@@ -138,81 +136,105 @@
+                  <% if (step.text) { %>
+                    <div id="info<%= scenarioIndex %><%= stepIndex %>-text" class="scenario-step-collapse collapse">
+                      <pre class=info><% try { %>
 <%= JSON.stringify(step.text, undefined, 2) %>
 <% } catch (error) { %>
 <%= step.text %>
 <% } %></pre>
-            </div>
-            <% } %>
+                    </div>
+                    <% } %>
 
-            <% if (step.image) { %>
-            <div id="info<%= scenarioIndex %><%= stepIndex %>-image" class="scenario-step-collapse collapse">
-                <a class="screenshot" href="<%= step.image %>" target="_blank">
+                      <% if (step.image) { %>
+                        <div id="info<%= scenarioIndex %><%= stepIndex %>-image" class="scenario-step-collapse collapse">
+                          <a class="screenshot" href="<%= step.image %>" target="_blank">
                     <img class="screenshot" style="height:100%;width:100%;" id="my_images" src="<%= step.image %>"/>
                 </a>
-            </div>
-            <% } %>
-            <% } %>
-
-            <% if(step.arguments) { %>
-            <% for( var i = 0; i < step.arguments.length; i++ ) { %>
-            <div class="scenario-scroll-bar">
-                <% if(step.arguments[i]["rows"]) { %>
-                <table class="table arguments">
-                    <% var rows = step.arguments[i]["rows"]; %>
-                    <tr>
-                        <% var cells = rows[0]["cells"]; %>
-                        <% for( var k = 0; k < cells.length; k++ ) { %>
-                        <th><%= cells[k] %></th>
-                        <% } %>
-                    </tr>
-                    <% for( var j = 1; j < rows.length; j++ ) { %>
-                    <tr>
-                        <% var cells = rows[j]["cells"]; %>
-                        <% for( var k = 0; k < cells.length; k++ ) { %>
-                        <td><%= cells[k] %></td>
+                        </div>
                         <% } %>
-                    </tr>
-                    <% } %>
-                </table>
-                <% } %>
-            </div>
-            <% if(step.arguments[i]["content"]) { %>
-            <pre class="info"><%= step.arguments[i]["content"].replace(/</g, '(').replace(/>/g, ')') %></pre>
-            <% } %>
-            <% } %>
-            <% } %>
 
-            <% } %>
-            <% }); %>
-        </div>
+
+                          <% if (!_.isEmpty(step.attachments)) { %>
+                            <% _.each(step.attachments, function(attachment, attachmentIndex) { %>
+                              <div id="info<%= scenarioIndex %><%= stepIndex %>-attachment<%= attachmentIndex %>" class="scenario-step-collapse collapse">
+                                <object class="attachment" style="height:100%;width:100%;" type="<%= attachment.type %>" id="my_attachments_<%= attachmentIndex %>" data="<%= attachment.data %>">
```

Comment:
Can we do something with this. I don't like the height, I'd prefer it to have the height of the content to get a good overview of the attachment, but I got this

![image](https://user-images.githubusercontent.com/11979740/36580528-eabc6088-1868-11e8-8622-bf85c4c5f2db.png)

Reply:
Yes, with chrome is a problem, with firefox removing style (copy and paste from image, sorry) appeare better, but is not a real solution. I try to search around internet if someone have a solution.

Reply:
Removing style we have
chrome: 
<img width="1198" alt="chrome" src="https://user-images.githubusercontent.com/226824/36614612-8e56da06-18dd-11e8-9763-1b00e9f0b7e8.png">

firefox:
<img width="1183" alt="firefox" src="https://user-images.githubusercontent.com/226824/36614613-8e75c4de-18dd-11e8-9131-3f30320b75df.png">

With embedding data the only solution acceptable seem to be this.
Seem to work well with audio too. 

I have try to use iframe instead of object/embed but chrome doesn't like.

I have evaluated as solution use PDF.js but the view increase his complexity and is specific for pdf.